src/amo/components/ServerHtml/index.js

```diff
@@ -122,17 +122,20 @@ export default class ServerHtml extends Component {
 
   getFontPreload() {
     const { assets, htmlLang } = this.props;
-    // Preload relevant minimal font if available. At the moment only for en-US.
-    if (htmlLang === 'en-US') {
-      const fontToPreload =
-        './src/fonts/woff2/Inter-roman-minimal-subset.var.woff2';
-      return this.getStatic({
-        filePath: assets.assets[fontToPreload],
-        type: 'font',
-        index: 0,
-      });
-    }
-    return null;
+    // Preload relevant minimal subset font if available for this language.
+    const extractSubset = /subset-([\w\-+]+)\.var\.woff2/;
+    return Object.keys(assets.assets)
+      .filter((asset) => {
+        const found = extractSubset.exec(asset);
+        return found && found.length === 2 && found[1].includes(htmlLang);
+      })
```

Comment:
```suggestion
    const subsetFontPattern = new RegExp(`subset-${htmlLang}.var.woff2$`);
    
    return Object.keys(assets.assets)
      .filter((asset) => subsetFontPattern.test(asset))
```

Reply:
(a bit more efficient and readable)

Reply:
The reason I'm doing it like this is to support things like `subset-fr+es+pt-BR` (which are likely to happen in the follow-up I filed)

Reply:
but it does not happen right now, right?

Reply:
No, but that's the very next thing I will be working on, so it didn't make much sense to simplify the code now in my mind (I've created the follow-up branch already, and tested that this approach will work there)

Reply:
but you could sill the regexp, no? Something like this:

```
new RegExp(`subset-([a-zA-Z\-]+\\+)*${htmlLang}(\\+[a-z\-]+)*.var.woff2$`, 'i')
```

Reply:
yeah, I'll do that.

Reply:
6a8abe15197e9eeef5f95b9cc5f15bdf2f23a702